src/utils/curriculum/formatting.ts

```diff
@@ -47,6 +47,35 @@ export function getPhaseText(
   return "";
 }
 
+export function getShortPhaseText(
```

Comment:
Yeah, I guess that there are only 4 possible variables (and only 2 can be used at a time), so making it more explicit can't really hurt, right?

Reply:
Yeah plan is to use single well tested functions around the place. Rather than repeating the logic about the codebase. There is now `getPhaseText(...)` (used in multiple places) and `getShortPhaseText(...)` currently used in 1 place.